source/tool/chipsec/helper/linux/helper.py

```diff
@@ -137,11 +139,35 @@ def init(self, start_driver):
 
             self._ioctl_base = fcntl.ioctl(self.dev_fh, IOCTL_BASE) << 4
 
+    def fallback_available(self):
```

Comment:
naming: fallback_available seems an awkward name. Will this conceivably be used for other mechanisms? Why not just devmem_available?